pkg/meta/model/index.go

```diff
@@ -285,6 +285,12 @@ func (index *IndexInfo) IsColumnarIndex() bool {
 	return index.VectorInfo != nil || index.InvertedInfo != nil || index.FullTextInfo != nil
 }
 
+// IsFulltextIndex checks whether the index is a fulltext index.
+// Fulltext index only exists in TiCI, no actual index data need to be written to KV layer.
+func (*IndexInfo) IsFulltextIndex() bool {
+	return false
```

Comment:
Why always return false?

Reply:
I don't know how to distinguish the fulltext search between global and tiflash local index. This PR is about tici, so I make it always returns false. Sure, check index.FullTextInfo! = nil is also ok.
addressed.